public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
remove all the comments from this file to clean it up a bit. Comments often pollute code and generally should not be checked in unless they are adding significant value. Trust that if code is well written you can quickly glance at it and know what it's doing.